src/app/workflow/workflow.component.html

```diff
@@ -171,7 +171,7 @@ <h3>
           class="private-btn small-btn-structure ml-2"
           matTooltip="Manage DOIs"
           (click)="manageDois()"
-          [disabled]="!canWrite"
+          [disabled]="!canWrite || (isRefreshing$ | async)"
```

Comment:
I noticed this button is not disabled when the page is loading after an action (ex: Publish) so I just added this. Same thing for the Manage labels button below